### PR TITLE
Makefile changes

### PR DESCRIPTION
**Pull request by oterral**
Some minor changes:

  - Adding global variable for bin files
  - Symplify lessc process

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -380,64 +380,58 @@ prd/checker: src/checker
...
 
-src/style/app.css: src/style/app.less \
-	    src/style/print.less \
-	    src/style/offline.less \
-	    src/style/ga_bootstrap.less \
-	    src/style/ga_variables.less \
-	    $(SRC_COMPONENTS_LESS_FILES) \
-	    node_modules
-	node_modules/.bin/lessc $(LESS_PARAMETERS) $< $@
+src/style/app.css: $(SRC_LESS_FILES)
```
Comment (loicgasser) [inline Makefile]: node_modules dependency disappeared. I am not sure this is wanted.

**Comment by loicgasser** [comment]
+1 for this, one comment otherwise nice improvement.

File: Makefile (hunk 1 above)
Comment (oterral) [inline Makefile]: since there is none less file dependent of a node module. I think we don't need it.

Comment (oterral) [inline Makefile]: and lessc node module is added when you do a `make all` the first time.

**Pull request merged**